hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -213,11 +213,13 @@ def _update_calendar_event(self):
             if rec.meeting_id:
                 rec.meeting_id.write(rec._prepare_holidays_meeting_values())
 
-    @api.model
+    @api.model_create_multi
+    @api.returns('self', lambda value: value.id)
     def create(self, values):
```

Comment:
Follow super argument name:
```suggestion
    def create(self, vals_list):
```